python/ray/serve/_private/http_proxy.py

```diff
@@ -409,11 +410,12 @@ def _ongoing_requests_end(self):
         self._ongoing_requests -= 1
         self.num_ongoing_requests_gauge.set(self._ongoing_requests)
 
-    async def __call__(self, scope, receive, send):
-        """Implements the ASGI protocol.
+    async def proxy_request(self, scope, receive, send):
+        """Wrapper for proxy request.
 
-        See details at:
-            https://asgi.readthedocs.io/en/latest/specs/index.html.
+        This method is served as common entry point by the proxy. It handles the
+        routing, including `/-/routes` and `/-/healthz`, ongoing request counter,
+        and metrics.
```

Comment:
the paths in the docstring are http-specifc

also a lot of these methods take `scope, receive, send` which are all http-specific

Reply:
correct, I want this first PR has no logical changes, just reorganize the methods, so we can get this merged in and I will raise the following PR to refactor using the new data models